site/source/cookbook/modern-web-apps/efficient-state-management.md

```diff
@@ -0,0 +1,67 @@
+---
+layout: docs
+category: modern-web-apps
+title: Manage state in your application
+overview: Manage state to decrease complexity in your application
+---
+
+## Objective
+
+This recipe demonstrates a technique for managing state in your Dojo 2 application.
+
+## Procedure
+
+1. Create a widget which uses a property from `this.properties`:
+
+```js
+render() {
+    return v('div', [this.properties.message]);
```

Comment:
Should our examples conform to the dojo2 style guide? If so then it should be `[ this.properties.message ]`

Reply:
Done! Maybe we could do automated linting for code snippets in our markdown files at some point